deployments/cloudfoundry/runtime-config-tile/tile.yml

```diff
@@ -0,0 +1,66 @@
+---
+name: signalfx-agent-runtime-config
+icon_file: ../bosh/resources/icon.png
+label: 'SignalFx Agent Runtime Config (BOSH Addon)'
+description: |
+  This deploys the SignalFx Smart Agent BOSH release to all deployments in the cluster.
+
+  Note that this depends on having the SignalFx Monitoring tile installed.
+
+forms:
+- name: agent_form
+  label: 'Agent Config'
+  description: 'Configuration of the Smart Agent on each host'
+  properties:
+  - name: agent_yaml
+    type: text
+    label: Agent Configuration
+    markdown: |
+      This is the [agent config
+      file](https://github.com/signalfx/signalfx-agent/blob/master/docs/config-schema.md)
+      to use.  This gives you maximum flexibility in configuring the agent.
+
+  - name: http_proxy
+    type: string
+    label: HTTP Proxy URL
+    description: If you need HTTP proxying, you can specify the proxy for HTTP traffic here
+    optional: true
+
+  - name: https_proxy
+    type: string
+    label: HTTPS Proxy URL
+    description: If you need HTTPS proxying, you can specify the proxy for HTTPS traffic here
+    optional: true
+
+  - name: no_proxy
+    type: string
+    label: Proxy Exclusions
+    description: If some domain names should not go through the proxy, specify them here as a comma-separated list.  Do not specify the scheme (e.g. http/https), only the domain names.
+    optional: true
+
+packages: []
+
+runtime_configs:
+  - name: signalfx-agent
+    runtime_config:
+      releases:
+        - name: signalfx-agent
+          version: 4.19.0-dev18
```

Comment:
is this the right version we want to point to?

Reply:
I have to merge this first and do a release of the agent with the firehose nozzle, and then I can fix up all of the release versions.